tests/unittests/unittests.cpp

```diff
@@ -42,6 +43,18 @@ struct Foo
 };
 
 
+class Wrapper
```

Comment:
Maybe a more descriptive name like `IntPtrWrapper` would be better?

Reply:
I changed it to UniquePtrWrapper.